### ISSUE TITLE
The burger menu doesn't work for clarity datagrid under IE10 when the grid is configured with single row selection

### ISSUE THREAD

**Issue by MirakoMachine** [vmware/clarity#2721]
## Describe the bug

In IE10 when you have a datagrid with single selection and single action as well - the burger menu works only when double clicking the button (single clicks don't work). 

## How to reproduce

1.  Have a grid with single selection and single action
2.  Open the page where the grid is in IE10
3.  Single click on the burger menu for any item in the grid
4.  The menu with action items does not show

## Expected behavior

The menu with action items should show when the burger button is clicked. 

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: Desktop
* OS: Windows
* Browser: Internet Explorer
* Version: 10

## Additional notes

Double clicking on the burger button will show the actions menu.

**Comment by youdz** [comment]
We actually do not support IE10, we start at IE11.

That said, if you reproduced your issue in a plunker, maybe we could find some time to help you out and see if we can identify the issue. For instance, are you using `[clDgRowSelection]="true"` by any chance? Because that could explain the issue you see.

**Comment by MirakoMachine** [comment]
Yeah `clDgRowSelection` is set to `true`. I think that causes the issue because it seams that the whole row is highlighted (only on IE10 again) when the burger menu is clicked.
